.gitignore

```diff
@@ -1,3 +1,6 @@
 .DS_Store
 erln8.config
 config.yml
+.bundle
+vendor
+Gemfile.lock
```

Comment:
Unless you're willing to fix everything that breaks every time something gets checked into chef/chef-zero (or any other dependent gem), please do not remove the Gemfile.lock from a piece of software that we use to run integration tests (both in CI and on-site with customers).

What benefit do we get from unlocking the gems? There's a very good reason there's a Gemfile.lock in this repo.